lib/jeckle/rest_actions.rb

```diff
@@ -5,19 +5,104 @@ def self.included(base)
     end
 
     module Collection
+      # @public
+      #
+      # The root name that Jeckle will parse the <b>response</b>. Default is <b>false</b>.
+      #
+      # @example
+      #
+      #   module Dribble
+      #     class Shot
+      #       include Jeckle::Resource
+      #       root collection: true, member: true
+      #     end
+      #   end
+      #
+      #   Shot.collection_root_name # => Will parse the root node as 'shots'
+      #   Shot.member_root_name # => Will parse the root node as 'shot'
+      #
+      #  Sometimes that are APIs you need to fetch /projects, BUT the root node is extremely different from the resource name.
+      #
+      #   module OtherApi
+      #     class Project
+      #       include Jeckle::Resource
+      #       api :my_api
+      #       root collection: 'awesome-projects', member: 'awesome-project'
+      #     end
+      #   end
+      #
+      def root(options={})
+        @collection_root_name = find_root_name(options[:collection], :pluralize)
+        @member_root_name     = find_root_name(options[:member], :singularize)
```

Comment:
@tomas-stefano I think these `root` definitions should be part of the `restful_action` DSL.

I'm working on something, will have something done soon here: #51